### ISSUE TITLE
AzurLane Crosswave's 3D Texures Missing 

### ISSUE THREAD

**Issue by 9H9P9D** [yuzu-emu/yuzu#6148]
There seems to be this bug with both Yuzu 582 and Yuzu EA 1559. Both versions fail to load in any kind of 3d textures but the game itself can still be played otherwise with no crashes as far as I can tell. 

![582-1](https://user-images.githubusercontent.com/81955166/113555966-befaba80-95c9-11eb-96bc-90421b07d601.png)

![582-2](https://user-images.githubusercontent.com/81955166/113555977-c28e4180-95c9-11eb-8329-ab12ef296b8b.png)

![1559](https://user-images.githubusercontent.com/81955166/113555986-c7eb8c00-95c9-11eb-9a63-593d1126c43c.png)

So far, this is the only bug I see for this game since everything else shows up perfectly (menus, reticles, character art, etc.) and no crashes from progressing through the game despite no 3d textures. 

tl;dr 2d shows properly. 3d doesn't show up at all.
